manager/src/main/scala/org/broadinstitute/transporter/kafka/KafkaClient.scala

```diff
@@ -84,6 +167,21 @@ object KafkaClient {
       }
   }
 
+  /**
+    * Summary of the results of an unsuccessful topic-creation request.
+    *
+    * @param failures failures encountered during the request, keyed by
+    *                 the corresponding topic name
+    * @param successes topics which were successfully created as part
+    *                  of the request
+    */
+  case class TopicCreationFailed(
+    failures: List[(String, Throwable)],
+    successes: Set[String]
```

Comment:
Any reason for this case class to contain successes as well?

Reply:
So that we know which topics to clean up when rolling things back. I'll add to the scaladoc.